### PR TITLE
[RFR] first-page poc

### PR DESCRIPTION
**Pull request by JulienMattiussi**
https://trello.com/c/sSmiLBOh

Globaly, site and environment is working
(Only main page, no "game" yet)

Lots of restructuration in next PR #3 
(Like tests, cleaning in Makefile)

### CODE REVIEW

**Review by alexisjanvier — CHANGES_REQUESTED** [review]
There is too much files for review ... Maybe did you add the `vendor` folder content before adding it to the `.gitignore`?

**Comment by Kmaschta** [comment]
Can you please not commit the vendor folder .. ?

File: Makefile [hunk 1]
```diff
@@ -25,8 +24,8 @@ stop: ## Stop the server
 	docker-compose down
 
 test: ## Test the code
-	docker build -t service_php docker/php
+	docker build -t php docker/php
```
Comment (alexisjanvier) [inline Makefile:27]: It's easier to use your existing services.

File: docker/php/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
 
-WORKDIR /quarto
\ No newline at end of file
+RUN apt-get update && apt-get install -y \
+        curl git zip unzip zlib1g-dev && \
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer && \
+    docker-php-ext-install -j$(nproc) zip
+
+# Install Postgre PDO
+RUN apt-get install -y libpq-dev \
```
Comment (alexisjanvier) [inline docker/php/Dockerfile:9]: This installation could be done at the same time as the first `RUN`

File: quarto/config/packages/doctrine.yaml [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
-        charset: utf8mb4
-        default_table_options:
-            charset: utf8mb4
-            collate: utf8mb4_unicode_ci
-
-        url: '%env(resolve:DATABASE_URL)%'
+        driver: 'pdo_pgsql'
+        host: 'postgres'
+        port: 5432
+        dbname: 'quarto'
```
Comment (alexisjanvier) [inline quarto/config/packages/doctrine.yaml:7]: This information should come from environment variables

File: quarto/src/Controller/MainController.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace App\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
```
Comment (alexisjanvier) [inline quarto/src/Controller/MainController.php:4]: Did you install a Php linter? for example to follow the [PSR-2](https://www.php-fig.org/psr/psr-2/) standard. Someting like [PHP Coding Standards Fixer](http://cs.sensiolabs.org/) or [PHP Formatter](https://marketplace.visualstudio.com/items?itemName=Sophisticode.php-formatter).

Here for example it seems to me that an empty line is missing.

**Review by alexisjanvier — CHANGES_REQUESTED** [review]
Could you add a screenshot?

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**